.github/contributing/commits.md

```diff
@@ -2,7 +2,9 @@
 
```

Comment:
Should we remove this file from the x-components package? There are duplicated files in the contributing folder of x-components: `base-naming.md`, `commits.md`,` components.md`,` tests.md`

Reply:
removed